hnn_core/cell.py

```diff
@@ -244,8 +218,8 @@ def syn_create(self, secloc, e, tau1, tau2):
 
         Parameters
         ----------
-        secloc : float (0. to 1.0)
-            The section location
+        secloc : instance of nrn.Segment
+            The section location. E.g., soma(0.5).
```

Comment:
Describing nrn objects with floats and ints shows up in several docstrings. Is this just referencing the numerical argument that instantiates the object?

Reply:
it might be documented wrongly :-) fix is always welcome!

btw, `soma(0.5)` just means the middle of the `soma` ... I guess you know that already but putting it out there in case that was not clear.